CondTools/Geometry/test/writehelpers/splitExtended2026Database.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-conddb_import -f sqlite_file:myfile.db -c sqlite_file:GeometryFileExtended2026D41.db -t XMLFILE_Geometry_TagXX_Extended2026D41_mc -i XMLFILE_Geometry_TagXX_Extended2026D41_mc
+conddb_import -f sqlite_file:myfile.db -c sqlite_file:GeometryFileExtended2026D49.db -t XMLFILE_Geometry_TagXX_Extended2026D49_mc -i XMLFILE_Geometry_TagXX_Extended2026D49_mc
```

Comment:
All the `conddb_import` lines should be replaced with `conddb`, as in `splitExtended2021Database.sh`. `conddb_import` is deprecated.

Reply:
done